enhancements/sig-architecture/141-grpc-based-registration/README.md

```diff
@@ -163,6 +163,132 @@ The following security principles should be considered between the broker and so
 - The sources should be authorized by broker to avoid one source can consume event messages from other sources
 - The agent should be authorized by broker to avoid one agent can consume event messages from other clusters 
 
+### Metrics
+
+The gRPC server exposes Prometheus metrics to monitor the health and performance. They are grouped into two categories:
+
+1. **General gRPC server metrics**
+2. **CloudEvents-specific gRPC server metrics**
+
+#### General gRPC server metrics
+
+Common metrics for gRPC server health and performance, started with `grpc_server` as Prometheus subsystem name.
+
+- **`grpc_server_active_connections`** (Gauge) - Current number of active connections.
+  For example:
+  ```
+  grpc_server_active_connections{local_addr="10.244.0.18:8090",remote_addr="10.244.0.16:45128"} 1
+  ```
+
+- **`grpc_server_started_total`** (Counter) - Total number of RPCs started on the server.
+  For example:
+  ```
+  grpc_server_started_total{grpc_method="Publish",grpc_service="io.cloudevents.v1.CloudEventService",grpc_type="unary"} 3
+  grpc_server_started_total{grpc_method="Subscribe",grpc_service="io.cloudevents.v1.CloudEventService",grpc_type="server_stream"} 4
```

Comment:
for `grpc_method="Subscribe",grpc_service="io.cloudevents.v1.CloudEventService"`,  we are able to get them from `StreamServerInfo`/`UnaryServerInfo`? so the method or service will not only cloudevents Publish/Subscribe, right?